tcms/rpc/api/testexecution.py

```diff
@@ -76,6 +76,23 @@ def remove_comment(execution_id, comment_id=None):
     to_be_deleted.delete()
 
 
+@permissions_required('django_comments.view_comment')
+@rpc_method(name='TestExecution.get_comments')
+def get_comments(execution_id):
```

Comment:
You still have pylint issues for this method.

Reply:
should be ok now